soh/soh/Enhancements/cosmetics/CosmeticsEditor.cpp

```diff
@@ -8,51 +8,62 @@
 #include <random>
 #include <math.h>
 #include <algorithm>
-#include <libultraship/libultra/types.h>
-#include "soh/Enhancements/randomizer/3drando/random.hpp"
 #include <libultraship/libultraship.h>
 
 #include "soh/UIWidgets.hpp"
 #include "soh/OTRGlobals.h"
 #include "soh/ResourceManagerHelpers.h"
 
 extern "C" {
-#include <z64.h>
-#include "macros.h"
-#include "soh/cvar_prefixes.h"
-extern PlayState* gPlayState;
-#include "objects/object_link_boy/object_link_boy.h"
-#include "objects/object_link_child/object_link_child.h"
-#include "objects/object_gi_shield_3/object_gi_shield_3.h"
-#include "objects/object_gi_heart/object_gi_heart.h"
-#include "objects/object_gi_bow/object_gi_bow.h"
-#include "objects/object_gi_bracelet/object_gi_bracelet.h"
-#include "objects/object_gi_rupy/object_gi_rupy.h"
-#include "objects/object_gi_magicpot/object_gi_magicpot.h"
-#include "objects/object_gi_gloves/object_gi_gloves.h"
-#include "objects/object_gi_hammer/object_gi_hammer.h"
-#include "objects/object_gi_sutaru/object_gi_sutaru.h"
-#include "objects/object_st/object_st.h"
-#include "objects/object_gi_boomerang/object_gi_boomerang.h"
-#include "objects/object_gi_liquid/object_gi_liquid.h"
-#include "objects/object_gi_bow/object_gi_bow.h"
-#include "objects/object_gi_hearts/object_gi_hearts.h"
-#include "objects/gameplay_keep/gameplay_keep.h"
-#include "objects/object_gi_sword_1/object_gi_sword_1.h"
-#include "objects/object_gi_longsword/object_gi_longsword.h"
-#include "objects/object_gi_clothes/object_gi_clothes.h"
-#include "objects/object_gi_bomb_2/object_gi_bomb_2.h"
-#include "objects/object_gla/object_gla.h"
-#include "objects/object_toki_objects/object_toki_objects.h"
-#include "objects/object_gi_pachinko/object_gi_pachinko.h"
-#include "objects/object_trap/object_trap.h"
-#include "overlays/ovl_Boss_Ganon2/ovl_Boss_Ganon2.h"
-#include "objects/object_gjyo_objects/object_gjyo_objects.h"
-#include "textures/nintendo_rogo_static/nintendo_rogo_static.h"
-#include "objects/object_gi_rabit_mask/object_gi_rabit_mask.h"
-u8 Randomizer_GetSettingValue(RandomizerSettingKey randoSettingKey);
+    #include "z64.h"
```

Comment:
This type of indentation is against the clang format rules we use. Formatting is not enforced yet, but lets not make this kind of change since we know it will just get reverted.